### DISCUSSION TITLE
Kong for k8s in GKE - OpenId-connect plugin

### DISCUSSION THREAD

**Discussion by NiranjanV12** [Kong/kong#7686] [Help]
Hi team,

I am trying to get kong for kubernetes running in **GKE**
I am getting this below error in ingress-controller logs for **openId-connect** plugin after `kubectl patch ingress`

```
time="2021-08-09T08:24:58Z" level=error msg="failed to fill-in defaults for plugin: openid-connect" component=controller
time="2021-08-09T08:24:58Z" level=error msg="failed to update kong configuration: posting new config to /config: HTTP status 400 (message: \"declarative config is invalid: {plugins={{name=\\\"plugin 'openid-connect' not enabled; add it to the 'plugins' configuration property\\\"}}}\")" component=controller
```

Please Suggest.

**Comment by kikito** [comment]
That error is saying that you are trying to use the `openid-connect` plugin but the `plugins` property in the Kong configuration does not include it. What is the value of your `plugins` property in the Kong configuration?
